tools/getWebpackConfig.js

```diff
@@ -30,10 +36,10 @@ module.exports = function getWebpackConfig(options) {
       libraryTarget: outputLibraryTarget || "umd",
       globalObject: "this",
     },
-    externals: {},
```

Comment:
first I was confused why remove it, then realized it was my mistake to have `externals` defined twice in this object 🙈 

thanks for cleaning it up!

Reply:
haha - it's so easy in webpack config files